### PR TITLE
QOL + Implemented Booster Endpoint

### PR DESCRIPTION
**Pull request by ghost**
### Added
- Added Boosters endpoint from [the api](https://docs.slothpixel.me/#tag/boosters)
- Created an example directory showing usage of the booster. This could also be used to demonstrate other endpoints.
I have provided the output of the usage below.
Example

### Changed
- I converted the build.gradle and settings.gradle to build.gralde.kts and settings.gradle.kts, It makes more sense as this library is written in kotlin
- Added /build and .idea to .gitignore

### Removed
- build.gralde + settings.gradle

### CODE REVIEW

File: src/main/kotlin/club/chachy/kpixel/utils/HttpExt.kt [hunk 1]
```diff
@@ -15,6 +15,7 @@ val http = HttpClient(Apache) {
 }
 
 suspend inline fun <reified T> HttpClient.getSlothpixelRoute(path: String, token: String?) =
-    get<T>("https://api.slothpixel.me/api$path") {
+    get<T>("https://api.slothpixel.me/api$path")
+    {
```
Comment (ChachyDev) [inline src/main/kotlin/club/chachy/kpixel/utils/HttpExt.kt]: why was this moved to a new line?

Comment (ghost) [reply src/main/kotlin/club/chachy/kpixel/utils/HttpExt.kt]: Accidentally woops

Comment (ChachyDev) [reply src/main/kotlin/club/chachy/kpixel/utils/HttpExt.kt]: ah okay

**Review by ChachyDev — APPROVED** [review]

**Comment by ChachyDev** [comment]
Merging
